libbeat/_meta/config.reference.yml

```diff
@@ -116,8 +116,8 @@
 #
 #   event -> filter1 -> event1 -> filter2 ->event2 ...
 #
-# The supported processors are drop_fields, drop_event, include_fields, and
-# add_cloud_metadata.
+# The supported processors are drop_fields, drop_event, include_fields, 
+# decode_json_fields and add_cloud_metadata.
```

Comment:
Could you please add a comma before the `and`. We usual go for the Oxford comma in our docs and config.

Reply:
Thank you! I'll take it.